buckwheat/tests/test_tokenizer.py

```diff
@@ -5,95 +5,149 @@
                          {"value": 15, "susp": 9, "println": 8, "lazy": 5, "some": 5, "maybe": 4,
                           "delay": 3, "none": 3, "int": 3, "lib": 2, "impl": 2, "option": 2,
                           "string": 2, "exampl": 1, "force": 1, "function": 1, "apply": 1,
-                          "tostr": 1, "evalu": 1, "main": 1, "args": 1, "array": 1}],
+                          "tostr": 1, "evalu": 1, "main": 1, "args": 1, "array": 1},
+                         ["// Contributed by John Williams\n", "/*", "*",
+                          " Delay the evaluation of an expression until it is needed. ",
+                          "*/", "/*", "*", " Get the value of a delayed expression. ", "*/",
+                          "/*", "*", " \n   ", "*", " Data type of suspended computations."
+                                                    " (The name froms from ML.) \n   ", "*/",
+                          "/*", "*", " \n   ", "*", " Implementation of suspended computations,"
+                                                    " separated from the \n   ", "*",
+                          " abstract class so that the type parameter can be invariant. \n   ",
+                          "*/", "// show that s is unevaluated\n", "// evaluate s\n",
+                          "// show that the value is saved\n", "// implicit call to force()\n",
+                          "// the type is covariant\n"]],
                         ["Shell", "test.sh",
                          {"echo": 10, "file": 8, "rip": 5, "exit": 4, "peaks": 3, "reads": 2,
                           "washu": 2, "root": 2, "tmpdir": 2, "which": 1, "cat": 1, "source": 1,
-                          "type": 1, "job": 1, "tmp": 1, "dir": 1, "mkdir": 1}],
+                          "type": 1, "job": 1, "tmp": 1, "dir": 1, "mkdir": 1},
+                         ["# If we already have rip file, do not recalculate"]],
                         ["Rust", "test.rs",
                          {"none": 6, "option": 4, "float": 3, "try": 2, "divis": 2, "println": 2,
-                          "unwrap": 2, "main": 1, "equival": 1, "some": 1}],
+                          "unwrap": 2, "main": 1, "equival": 1, "some": 1},
+                         ["// Binding `None` to a variable needs to be type annotated",
+                          "// Unwrapping a `Some` variant will extract the value wrapped.",
+                          "// Unwrapping a `None` variant will `panic!`"]],
                         ["Swift", "test.swift",
                          {"interfac": 11, "address": 8, "target": 5, "first": 2, "nio": 1,
                           "network": 1, "command": 1, "line": 1, "argument": 1, "drop": 1,
                           "socket": 1, "ipaddress": 1, "port": 1, "system": 1, "enumer": 1,
-                          "fatal": 1, "error": 1}],
+                          "fatal": 1, "error": 1},
+                         ["//", "/", " ", "D", "e", "f", "i", "n", "i", "n", "g", " ",
```

Comment:
this comment looks strange - does it mean that we can extract comments in swift only as a list of chars?

Reply:
Remove Swift from comment extraction supported languages.